src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.cs

```diff
@@ -75,13 +80,17 @@ private static bool IsEccAlgorithmGroup(CngAlgorithmGroup algorithmGroup)
         private CngAlgorithmCore _core;
 
         // See https://msdn.microsoft.com/en-us/library/windows/desktop/bb931354(v=vs.85).aspx
+        // See https://msdn.microsoft.com/en-us/library/windows/desktop/mt632245%28v=vs.85%29.aspx for named curves
         private static readonly KeySizes[] s_legalKeySizes =
             new KeySizes[]
             {
                 // All values are in bits.
-                new KeySizes(minSize: 256, maxSize: 384, skipSize: 128),
-                new KeySizes(minSize: 521, maxSize: 521, skipSize: 0),
+                new KeySizes(minSize: 160, maxSize: 224, skipSize: 32), // 160, 192, 224
```

Comment:
That was the exact same approach that I was doing for RSA, and the feedback was "why not just turn it off when changing the KeySize"?  I guess really it depends on what we think LegalKeySizes means.

A) These are the values you can send to set_KeySize and it will do something.
B) These are the sizes of keys this class can understand.

If (A) then it'll be the 3 previous curves and setting it to wide-open during assignment of the "what is the current KeySize" value.

If (B) then we'd just leave it wide open and make the setter restrictive.

So... I guess using RSA as precedent (RSACng uses it as the key sizes it can generate, but it can read outside that range) it's probably (B)?